src/ado2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -86,13 +86,13 @@ public async Task Handle(MigrateRepoCommandArgs args)
         {
             _log.LogError($"Migration Failed. Migration ID: {migrationId}");
             _warningsCountLogger.LogWarningsCount(warningsCount);
-            _log.LogInformation($"Migration log available at {migrationLogUrl} or by running `gh {CliContext.RootCommand} download-logs --github-target-org {args.GithubOrg} --target-repo {args.GithubRepo}`");
```

Comment:
Can we throw this into a variable? So that it only needs to be updated once?

Reply:
Done in a1e0897